shells/electron/src/backend.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+declare var FOR_BACKEND: {
+  wall: Object;
+  resolveRNStyle: () => void;
+};
+
+if (!window.performance) {
```

Comment:
ah yes. I copied that for here: https://github.com/iamdustan/react-hardware/blob/516d2a8854028577a7d7928729042c8738181bb0/src/devtools/setupDevtools.js